ZipBuilder/Sources/ZipBuilder/ModuleMapBuilder.swift

```diff
@@ -18,18 +18,16 @@ import Foundation
 
 /// A struct to build a .modulemap in a given framework directory.
 ///
-struct ModuleMapBuilder {
+class ModuleMapBuilder {
```

Comment:
Same question - what's the reason for making this a `class`?

Reply:
Actually not needed. Only the other one needs to be a class.